lib/conjur/base.rb

```diff
@@ -169,6 +169,13 @@ def initialize username, api_key, token, remote_ip = nil
       @username = username
       @api_key = api_key
       @token = token
+      
+      # Assume that the token was recently minted, log a warning if not
+      @token_born = gettime   
+      if token && (delta = Time.now - Time.parse(token['timestamp'])) > 10
+        $stderr.puts "WARNING: token timestamp offset by #{delta.to_i} seconds"
+      end
+
```

Comment:
@dividedmind can you review this, please? I don't really love this solution. Seems like it would be better to set `@token_born` based on `token['timestamp']`. But, the values from `gettime` and `Time.parse` aren't comparable, so I wasn't sure what else to do.